ui/app/css/itcss/components/newui-sections.scss

```diff
@@ -14,6 +14,47 @@ $sub-mid-size-breakpoint-range: "screen and (min-width: #{$break-large}) and (ma
   align-items: center;
 }
 
+// Fix for UI lag on external monitor: https://github.com/MetaMask/metamask-extension/issues/10173
+.app::after {
+  content: "";
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 1px;
+  height: 1px;
+  background-color: $Grey-000;
+  -webkit-animation-name: emptySpinningDiv;
```

Comment:
I suspect that these prefixed entries aren't required, because we use `gulp-autoprefixer` when building our styles.

Reply:
Spot on. I wasn't sure if autoprefixer was implemented. Found other `-webkit-` rules in other css files and decided to keep it.
Gonna update